site/controllers/university.js

```diff
@@ -1,33 +1,25 @@
 const City = require('../models/city');
 
-exports.postAddUniversity = (req, res, next) => {
+exports.postAddUniversity = async (req, res, next) => {
   const cityName = req.body.city;
-  const universityName = req.body.universityName
+  const universityName = req.body.university
 
-  City.findOne({ where: { cityName: cityName  } })
-    .then((city) => {
-      console.log(city);
-      if (city) {
-        city.createUniversity({ universityName });
-      } else {
-        const city = City.build({cityName})
+  const [city, created] = await City.findOrCreate({
+    where: { cityName: cityName }
+  })
+  city.createUniversity({ universityName })
+  
+  if (created) {
+    res.json('university and city created')
+  } else {
+    res.json('university created')
+  }
 
-        try {
-          await city.save();
-          city.createUniversity({ universityName });
-          res.send("City and University created succesfully");
-        } catch (err) {
-          res.status(400).send(err);
-        }
-      }
-    })
-    .then((results) => res.json("University is created succesfully"))
-    .catch((err) => console.log(err));
   };
 
-exports.getUniversities = (req, res, next) => {
+exports.getAddUniversity = (req, res, next) => {
```

Comment:
Again here about the naming convention, please use a more proper method name because this method isn't adding any university to the database.